### PR TITLE
English capitalization

### PR DESCRIPTION
**Pull request by dackst**
This attempts to improve the capitalization in English dialogue. For the most part, this was decapitalizing words like "today" that have no business being capitalized.

https://en.wikipedia.org/wiki/Proper_and_common_nouns#English_capitalization_of_proper_nouns

Cardinal directions like "northerner" are one of the "gray areas" mentioned in the above article. I tried to consistently capitalize the directions when used to refer to names or political regions (really only applicable to the Snow World and "East Street") while lowercasing other capitalized uses of them. For example, the spring is to the west of the first town, but not in the West.

As the capitalization of His/Her Majesty and people's titles (e.g. "the King") were all over the place, I briefly booted up some modern Dragon Quest games and saw they have them capitalized, even when not used as a name, so I went with that. Things like His/Her Majesty, [King](https://user-images.githubusercontent.com/35816592/115112474-b43e0f00-9f53-11eb-94a6-132d4bd42f93.png), [Prince](https://user-images.githubusercontent.com/35816592/115112427-88228e00-9f53-11eb-9ecc-69e191c5d8fa.png)(ss), [Chancellor](https://user-images.githubusercontent.com/35816592/115112449-9cff2180-9f53-11eb-8be8-65a36be9e23c.png), Ringmaster, [Mayor](https://user-images.githubusercontent.com/35816592/115111854-4d6b2680-9f50-11eb-8aa3-85af85bbb2ce.png), and Elder should now be more consistently capitalized.

I also noticed these Dragon Quest games decapitalize [generic](https://user-images.githubusercontent.com/35816592/115111970-29f4ab80-9f51-11eb-9600-8cdceaf8664a.png) [monster](https://user-images.githubusercontent.com/35816592/115112547-08e18a00-9f54-11eb-965f-4ddae929182a.png) names, and pluralize them like regular words. This differs from something like Pokemon, where every Pokemon name is always capitalized, and is treated as both singular and plural. Names of generic monster varieties should now be decapitalized in dialogue, while special monsters like "Stalagosaur" and "Woolly Elefort" should now be always capitalized in dialogue.

Uncapitalized item names were made always capitalized to match the messages when you find an item on the ground or in a chest. This [doesn't match other English games](https://user-images.githubusercontent.com/35816592/115112028-6aecc000-9f51-11eb-9be7-0f7f9b064759.png), but I suspect it might be more difficult to implement.

"Navel" was usually capitalized, so now it's always capitalized (when not referring to the literal navel of a person).

As "other world(s)" was already used more often than "Other World(s)", I decapitalized cases of "Other World". While I was at it, I also went ahead and tried to make it so it is no longer be used in ways that imply only one other world exists, like "the other world".

There are a few other cases where I very slightly adjusted other text nearby.

### CODE REVIEW

**Review by Ich73 — CHANGES_REQUESTED** [review]
Thanks a lot for your efforts! Most of your changes are good.

However we'd like you to change the following things:
 * `Event_EN\Field\AWA01`, line 92: revert this change, because Stalagosaur is its name (but also the name of the monster type)
 * `Event_EN\Field\C***`: revert all changes that are not capitalizations, because this "broken English" refers to Norden soldiers talking with a broken English / Russian accent

**Comment by dackst** [comment]
tl;dr: Let me know if you're sure about reverting them, and I'll move the non-capitalization reversions in my `capitalization2` branch to this one.

---

The Stelagosaur line is to coincide with the line right above it, "The Stalagosaur is on the move." There's many more occurrences of "the Stelagosaur", and a few occurrences of "the Woolly Elefort". I could see if they should __all__ be changed to drop the "the".

It also changes "will stop" to "has stopped". I think the previous version kind of implies it will continue to move for a time until it stops in the future, instead of simply stopping right away. 

The only reason I touched this line in the first place was because I initially decapitalized the name to "stelagosaur" before changing my mind.

---

For Norden, I noticed the townspeople are supposed to be speaking with an accent. I never claimed to "fix" such "broken English", and I don't think anything conflicts with it. I went through all the changes in the `C***` files again, and here's what should be all of the remaining non-capitalization changes:


> CDB01, line 21: followed suite -> followed suit
> CDB01,47: passed this point -> past this point
> CDB01,58 (and 60,62): But don't worry if we strike it rich we're going to divide it evenly. -> But don't worry. If we strike it rich, we're going to divide it evenly.
> CDB01,73: make mince meat -> make mincemeat
> CDB01,103 (105,107): smile back princess' -> smile back to Princess (while it could be an accent valid, accent Norden doesn't seem to use adjectives swapped)
> CDB01,109 (111,113): cheep up princess -> cheer up Princess
> CDB01,110 (112,114): meet the princess -> meet Princess (playing into Norden accent more from Norden soldier)
> 
> CDG01,17: blocked by the King's order who is worried -> blocked by order of the King, who is worried (original makes it sound like it is the order who is worried and not the king, which is obviously impossible. Also Nofor people don't seem to have the same accent, or else it would be "by King's order/order of King")
> CDG01,18 (19): You'll reach the port town of Nofor when you come out of this tunnel. -> You'll reach the port town of Nofor when you come out of this tunnel. (These lines are repeats of line 16 in Japanese without the [CONT], but for some reason the edited text is different?)
> 
> CTA01,55: You are the hero of Norden! -> You are hero of Norden! (playing into Norden accent again)
> CTA01,65 (67): monsters appeared near the spirit's spring. -> monsters appeared near Spirit Spring (how most people in Norden refer to the spring)
> 
> CTA04,27: When all of your party members defeated during your travels, you'll lose 1/2 of gold... -> When all of your party members are defeated during your travels, you'll lose 1/2 of your gold... (this should have been applied to all identical bank messages that come before the subsequent lines about Gold and the Field, and is not exclusive to the Snow World)
>   - also in CTD03,28
> 
> CTA08,34: wolrd -> world
> 
> CTA09,17: Speaker of the Spirit -> Speaker of Spirit (playing into Norden accent again, how most people in Norden refer to the Speaker)
> CTA09,26: his real Excellency -> real His Excellency (looks weird to have "His" capitalized when there's another word in the middle)
> CTA09,80: Westeranian -> Westanian
> 
> 
> CTB01,50: Truth be told my brother's a soldier in the west -> Truth be told, my brother's a soldier in the West.
> CTB01,51: Though he deserted the northern army he wants to come -> Though he deserted the Northern army, he wants to come
> CTB01,58: Well, kids like to play outside so I guess it's fine for a little while. -> Well, kids like to play outside, so I guess it's fine for a little while.
> 
> CTB02,36: But just between us I hear -> But just between us, I hear
> CTB05,18: So were we waging war against Westania, in the first place? -> So why were we waging war against Westania in the first place?
> CTB05,19: I can't believe they thought about going to war against the north and the west now. -> I can't believe the North thought about going to war against the West now.
> CTB05,21: behind the scene -> behind the scenes
> 
> CTC03,21: Wait does that mean I'm... -> Wait, does that mean I'm...
> CTC03,41: Ohhh, I screw up. -> Ohhh, I screwed up.
> 
> CTC04,25: hs -> his
> CTC04,64: tom boy -> tomboy
>    - also in CTE05, lines 18 and 61
> 
> CTE01,24 (26): between the Westania and Norden. -> between Westania and Norden.
> CTE05,36: I heard Prince Sonya who I taught as a student long ago was cursed -> I heard Prince Sonya, who I taught as a student long ago, was cursed
> 
> CTE06,34: THAt -> THAT
> CTE06,70: That crest is!? -> That crest!?
> 
> CTE07,21: town of Nofor south of Norden -> town of Nofor, south of Norden
> 
> CWA01,57: successfully Scouted the master of the Snow World, Wolly Elefort! ->  scouted the master of the Snow World, Wooly Elefort! (this should be more similar to the "defeated master" message, as well as the "scouted master" messages in the other worlds.)


I made most of these changes quickly while skimming through for cardinal directions or other possible words to (de)capitalize. I acknowledge the above changes are outside of the stated scope.

If I did everything correctly, all these lines (including Stalagosaur, not including capitalization changes or the bank line when used outside of `C***` files) should be reverted to the previous version on a different branch (`capitalization2`). Let me know if you want to continue with reverting them, and I'll move them over to this one.

**Comment by Ich73** [comment]
Thanks for your clarification. We've discussed again and all changes look fine.
I'll approve them now. Thanks again for your efforts.
